crates/bevy_diagnostic/src/lib.rs

```diff
@@ -1,3 +1,7 @@
+//! This crate provides a lightweight solution for computing diagnostics in the [Bevy game engine](https://bevyengine.org/).
+//! It allows users to easily add diagnostic functionality to their Bevy applications, enchancing
+//! their ability to monitor and optimize their game's without the need for additional dependencies.
```

Comment:
`bevy_diagnostic` is not particularly lightweight. it's also strange to mention no dependencies when this can be seen as an additional dependency.

if we consider that `bevy_diagnostic` is not a clear name, then I think "computing diagnostic" is even more confusing

Reply:
Agreed; we shouldn't use the same term to describe what this does.

"Straightforward" is maybe a better description than lightweight?